README.md

```diff
@@ -1,40 +1,121 @@
-# SPDX-Python-LicenseMatching
-Implement SPDX License Matching in Python. Project in CommunityBridge Linux Foundation 2020. 
-The Software Package Data Exchange (SPDX) specification is a standard format for communicating the components, licenses and copyrights associated with a software package. This Project is mainly based on implementing License Matching Techniques in Python. 
+# SPDX Python Library for LicenseMatching
+A Python Library to implement SPDX License Matching. This Project is a result of CommunityBridge Linux Foundation 2020 contribution by [@anshuldutt21](https://github.com/anshuldutt21/).
 
-## Check if 2 License Texts match
-License Texts are matched based on the [SPDX License Matching Guidelines](https://spdx.dev/license-list/matching-guidelines/). In order to implement License Matching between 2 License Texts, do the following steps.
+Python : Version>=3.7
+
+Home: https://github.com/anshuldutt21/spdx_python_licensematching/
+
+Issues: https://github.com/anshuldutt21/spdx_python_licensematching/issues
+
+# License
+[Apache-2.0](https://github.com/spdx/tools-python/blob/master/LICENSE)
+
+# Features
 
-Clone the Repository and move to the spdx-python-licensematching folder.
+*    An Interface which compares text against a license template using the license matching guidelines.
+*    An Interface which returns all matching SPDX listed license ID's for any license text.
+*    An interface which compares 2 license texts and returns a boolean indicating if the 2 licenses match per the license matching guidelines.
+*    When there is no match, a return value is provided to describe where and why the license does not match.
 
-Move the files in the normalize_license_text folder.
+# Installation
+
+As always you should work in a virtualenv or venv. Do the following steps to setup the Python Package in your system.
 
-Execute the command 
 ```bash
-Python normalize.py <File1> <File2>
+git clone https://github.com/anshuldutt21/spdx_python_licensematching.git
+cd spdx_python_licensematching
```

Comment:
Add a simple command before this to create virtual env
`virtualenv venv -p python3.7`

Reply:
@anshuldutt21 Add this.